api/controllers/transactions.js

```diff
@@ -44,7 +44,7 @@ function transactionFormatter(transaction) {
 	result.senderId = result.senderId || '';
 	result.recipientId = result.recipientId || '';
 	result.recipientPublicKey = result.recipientPublicKey || '';
-	result.signSignature = result.signSignature || '';
+	result.signSignature = result.signSignature || undefined;
```

Comment:
```suggestion
	if (!result.signSignature) {
		delete result.signSignature;
	}
```
Np?

Reply:
The suggestion has the same effect than the solution. Swagger takes care of not showing it when it is undefined.